cmake/Modules/FindComputeCpp.cmake

```diff
@@ -58,19 +58,29 @@ find_package(OpenCL REQUIRED)
 
 # Find ComputeCpp package
 
-# Try to read the environment variable
 if(DEFINED ComputeCpp_DIR)
   set(computecpp_find_hint ${ComputeCpp_DIR})
 elseif(DEFINED ENV{COMPUTECPP_DIR})
   set(computecpp_find_hint $ENV{COMPUTECPP_DIR})
 endif()
 
+if(CMAKE_CROSSCOMPILING)
```

Comment:
Argh, I asked you to put this stuff inside CMAKE_CROSSCOMPILING, but that breaks the script (your local test worked because of cache variables). You should initialise the computecpp_native_find_hint to computecpp_find_hint, then let it be mutated in the branch. My fault, sorry!